crates/symbolicator/src/services/symcaches.rs

```diff
@@ -450,7 +450,7 @@ mod tests {
     async fn test_symcache_refresh() {
         test::setup();
 
-        const TIMEOUT: Duration = Duration::from_millis(500);
+        const TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
```suggestion
        const TIMEOUT: Duration = Duration::from_secs(2);
```

The flackyness might be related to filesystem timestamps. I don’t think we have to be overly generous with the timeout.

Reply:
Hum, but what's wrong with being overly generous?  I'd prefer to give it plenty of time.  CI machines are busy sometimes.